### ISSUE TITLE
Cannot find module 'babelify'

### ISSUE THREAD

**Issue by winkler1** [kern/filepizza#17]
Getting this when hitting the page... `npm install -g babelify` did not help.

```
$ env PORT=8080 filepizza
FilePizza listening on :::8080
GET / 200 28.960 ms - -
GET /images/up.png 304 3.406 ms - -
GET /images/pizza.png 304 0.841 ms - -
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Cannot find module 'babelify' from '/Users'
    at /usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:50:17
    at process (/usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:119:43)
    at /usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:128:21
    at load (/usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:60:43)
    at /usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:66:22
    at /usr/local/lib/node_modules/filepizza/node_modules/browserify-middleware/node_modules/browserify/node_modules/resolve/lib/async.js:21:47
    at FSReqWrap.oncomplete (fs.js:99:15)```

**Comment by anubiann00b** [comment]
:+1:

**Comment by kern** [comment]
Seems to be some issue with older versions of babel or node/npm. Which versions of these do you have installed?

**Comment by winkler1** [comment]
Upgrading didn't fix it. `node -v` was v0.12.0. No global babel installed.

Did a full reinstall of node to 0.12.7, still getting same error.

**Comment by Boztown** [comment]
I'm getting this too (when I go to hit the server for the first time).  Node 0.12.6.  NPM 1.4.6.

**Comment by kern** [comment]
I'm unable to replicate this on Node 0.12.2 and NPM 2.10.0. It might be a problem with older versions of NPM or related to the old versions of browserify/babelify I was using. I've bumped the versions of all Browserify-related deps in 9a6b1b8. Could you check to see if master is working on your setup now?

**Comment by kern** [comment]
Should be fixed in `0.2.2`. I removed the runtime dependency on Babel, which seemed to be the issue.
